### PR TITLE
Problem: "accumulation()" function has no test cases #50

### LINKED ISSUE #50: Problem: "accumulation()" function has no test cases
Sub of #2
It can be proceeded after #45

### PR DESCRIPTION
**Pull request by CTAPCKPIM**

### CODE REVIEW

**Comment by CTAPCKPIM** [comment]
@accompany1205 
Help pls

 - TODO: 
**Solve the problem with the fact that the numbers do not converge**
_I have been thinking and trying different options for more than an hour, and without results_

**Adding other tests for this function**
_I'll do it myself but stalled over the above_

**Comment by CTAPCKPIM** [comment]
I think the problem is in lines **186, 187**
Like something at the stage when we rewind time
image

**Comment by CTAPCKPIM** [comment]
- Done, `accumulation();` has tests

**Comment by zolotokrylin** [comment]
sync with main branch 
tnx

**Review by zolotokrylin — CHANGES_REQUESTED** [review]
sync the branch (resolve conflicts)

**Comment by markholdex** [comment]
Accumulation function already has tests. Closing this PR

File: test/StreamManager.test.js [hunk 1]
```diff
@@ -164,8 +168,58 @@ describe("StreamManager", function () {
...
+    );
+
+    // Setting tinestamp
+    const currentTime = Math.floor(Date.now() / 1000)
+    await time.increase(2 * 24 * 3600); // + 2 days
+
+    // Calling the `accumulation();`
+    const accumulatedAmount = await this.streamManager.connect(this.payee1).accumulation()
+    // Calculating expected amount
+    const expectedAmount = Math.floor(this.cliffPeriod * this.amount / 30 / 24 / 3600)
```
Comment (markholdex) [inline test/StreamManager.test.js:192]: Not correctly calculated. During cliff period, 0 amount is accumulated. Check existing tests

**Pull request closed without merging**